horovod/spark/keras/remote.py

```diff
@@ -154,7 +157,11 @@ def train(serialized_model, train_rows, val_rows, avg_row_size):
                 if _checkpoint_callback:
                     _checkpoint_callback.filepath = ckpt_file
                 else:
-                    _checkpoint_callback = k.callbacks.ModelCheckpoint(ckpt_file)
+                    if is_dbfs and LooseVersion(tf.__version__) < LooseVersion("2.0.0"):
+                        _checkpoint_callback = k.callbacks.ModelCheckpoint(ckpt_file,
```

Comment:
would be nice to have an inline comment to explain why (DBFS local APIs does not support random write).